api/src/main/java/org/apache/iceberg/transforms/VoidTransform.java

```diff
@@ -24,7 +24,7 @@
 import org.apache.iceberg.expressions.UnboundPredicate;
 import org.apache.iceberg.types.Type;
 
-class VoidTransform<S> implements Transform<S, Void> {
+public class VoidTransform<S> implements Transform<S, Void> {
```

Comment:
Do we have to open this?

Reply:
I don't have a strong preference on keeping it private or opening it up. It can alternatively be done in this [way](https://github.com/apache/iceberg/pull/3059#discussion_r700591049). What's your thought?

Reply:
Anton and I discussed maybe adding a `isVoid` method on Transform much like we have `isIdentity`

```
  /**
   * Return whether this transform is the identity transform.
   *
   * @return true if this is an identity transform, false otherwise
   */
  default boolean isIdentity() {
    return false;
  
  ```